src/components/tables/DataExplorerTable/index.jsx

```diff
@@ -1,62 +1,16 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import DataTable from '@arranger/components/dist/DataTable';
+import { Table } from '@arranger/components/dist/Arranger';
 import '@arranger/components/public/themeStyles/beagle/beagle.css';
-import fileDownload from 'js-file-download';
```

Comment:
Are we going to get rid of `beagle.css`?

Reply:
PXD-1325